build/dependencies.props

```diff
@@ -46,7 +46,7 @@
     <MicrosoftExtensionsConfigurationPackageVersion>2.1.0-preview1-27474</MicrosoftExtensionsConfigurationPackageVersion>
     <MicrosoftExtensionsCopyOnWriteDictionarySourcesPackageVersion>2.1.0-preview1-27474</MicrosoftExtensionsCopyOnWriteDictionarySourcesPackageVersion>
     <MicrosoftExtensionsDependencyInjectionPackageVersion>2.1.0-preview1-27474</MicrosoftExtensionsDependencyInjectionPackageVersion>
-    <MicrosoftExtensionsDependencyModelPackageVersion>2.0.0</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>2.1.0-preview2-25711-01</MicrosoftExtensionsDependencyModelPackageVersion>
```

Comment:
This is troublesome isn't it? I'd imagine the automatic update would override this.

Reply:
I've updated the file in Universe to use this new version (https://github.com/aspnet/Universe/commit/14888f46bca342d29767197265b9f8652a5a0277). I'm getting this out in the meanwhile to unblock further work.